### PR TITLE
Notifications List component

### PR DESCRIPTION
**Pull request by knesko**
For [#3 ](https://github.com/antonpanchenk0/lms_system/issues/3) 

1. Created notification component
2. Created notifications list component
3. Added redux (actions, reducer)
4. Created InitComponent

### CODE REVIEW

File: frontend/src/global/styles/index.scss [hunk 1]
```diff
@@ -2,5 +2,5 @@
 body,
 htlm,
```
Comment (kuprienko0) [inline frontend/src/global/styles/index.scss:3]: change to html, please

**Review by yaroslavbilokin — APPROVED** [review]

**Comment by antonpanchenk0** [comment]
`frontend/src/pages/InitComponent/InitComponent.scss` are we need this file?

`frontend/package-lock.json` please remove this one from changes. If this file not included to `.gitignore`, please will add it.

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import InitComponent from './pages/InitComponent';
 import './global/styles/index.scss';
 
-const App = () => <div>Application</div>;
+const App = () => <InitComponent />;
```
Comment (antonpanchenk0) [inline frontend/src/App.jsx:5]: add this code here, please: 

```bash
import { BrowserRouter as Router, Switch } from 'react-router-dom'

const App = () => (
<Router>
  <Switch>
    <InitComponent />
  </Switch>
</Router>
)
```

File: frontend/src/pages/InitComponent/components/NotificationList/Notification/Notification.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+const Notification = ({ notification: { id, type, content, canBeClosed }, onClose }) => {
+  useEffect(() => {
+    if (type !== NOTIFICATIONS_CONFIG.type.error) {
+      const timeout = setTimeout(() => {
+        onClose(id);
+      }, 5000);
+      return timeout;
+    }
+    return null;
```
Comment (antonpanchenk0) [inline frontend/src/pages/InitComponent/components/NotificationList/Notification/Notification.jsx:15]: We need to clear interval when component will unmount

File: frontend/src/pages/InitComponent/components/NotificationList/Notification/Notification.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import { NOTIFICATIONS_CONFIG } from '../../../сonstants';
+import './Notification.scss';
+
+const Notification = ({ notification: { id, type, content, canBeClosed }, onClose }) => {
+  useEffect(() => {
+    if (type !== NOTIFICATIONS_CONFIG.type.error) {
+      const timeout = setTimeout(() => {
+        onClose(id);
+      }, 5000);
+      return timeout;
```
Comment (antonpanchenk0) [inline frontend/src/pages/InitComponent/components/NotificationList/Notification/Notification.jsx:13]: I think this return is not necessary.

**Review by antonpanchenk0 — CHANGES_REQUESTED** [review]

**Review by antonpanchenk0 — APPROVED** [review]

**Review by yaroslavbilokin — APPROVED** [review]

**Pull request merged**